packages/web/pages/contact-us.js

```diff
@@ -0,0 +1,206 @@
+	const handleReset = () => {
+		const { reset } = form;
+		reset();
+	};
+
+	return (
+		<StyledActions>
+			<Button type="submit" variant="contained">
+				Enviar mensagem
+			</Button>
+
+			<CancelButton type="button" variant="outlined" onClick={handleReset}>
+				Limpar campos
+			</CancelButton>
+		</StyledActions>
+	);
+};
+
+Actions.propTypes = {
+	form: PropTypes.shape({
+		reset: PropTypes.func,
+	}),
+};
+
+Actions.defaultProps = {
+	form: {},
+};
+
+const inputWrapperCss = css`
+	margin-top: 1.6rem;
+`;
+
+const ContactUs = () => {
+	const { openModal } = useModal();
+
+	const handleSubmit = () => {
+		openModal('contactUsSuccess');
+	};
+
+	return (
+		<Wrapper>
+			<Container>
+				<Form onSubmit={handleSubmit} defaultValues={{ user: { contactReason: '' } }}>
+					<StyledTitle align="left" noPadding noMargin>
+						Fale conosco
+					</StyledTitle>
+					<InputField
+						name="user.name"
+						type="text"
+						label="Qual o seu nome?"
+						validation={{ required: true }}
+						variant="gray"
+					/>
+
+					<InputField
+						name="user.email"
+						type="email"
+						label="Seu melhor e-mail"
+						validation={{ required: true }}
+						wrapperCss={inputWrapperCss}
+						variant="gray"
+					/>
+
+					<InputField
+						name="user.phone"
+						type="text"
+						label="Telefone"
+						wrapperCss={inputWrapperCss}
+						variant="gray"
+					/>
+
+					<SelectField
+						instanceId="contact-form-reason"
+						name="user.contactReason"
+						label="Qual o motivo do contato?"
+						placeholder="Escolha um assunto"
+						options={contactOptions}
+						validation={{ required: true }}
+						wrapperCss={inputWrapperCss}
+						variant="rounded"
+					/>
+
+					<TextField
+						name="user.message"
+						label="Sua mensagem"
+						validation={{ required: true }}
+						wrapperCss={inputWrapperCss}
+						variant="gray"
+					/>
+
+					<Actions />
+				</Form>
+			</Container>
+			<img
+				src="waves-rafiki-contact.svg"
+				alt="Ilustração de uma moça sentada em um sofa verde mexendo em um notebook com uma ilustração de ondas ao fundo"
+			/>
+		</Wrapper>
+	);
+};
+
+const Wrapper = styled.section`
+	${({ theme: { screens } }) => css`
+		position: relative;
+		overflow: hidden;
+		display: grid;
+		grid-template-columns: repeat(2, 50%);
+		padding: 0 1.6rem;
+		margin-top: 0.2rem;
+
+		> img {
+			position: absolute;
+			bottom: 0;
+			right: 0;
+			z-index: 1;
+			width: 100%;
+			max-width: 128rem;
+		}
+
+		@media screen and (max-width: ${screens.large}px) {
+			grid-template-columns: repeat(1, 100%);
+
+			> div {
+				max-width: 80rem;
+			}
+
+			> img {
+				display: none;
+			}
+		}
+	`}
+`;
+
+const Container = styled.div`
+	display: flex;
+	margin: 0 auto;
+	max-width: 57rem;
+	width: 100%;
+	position: relative;
+	z-index: 2;
+
+	> img {
+		margin-left: 8.8rem;
+	}
+`;
+
+const StyledTitle = styled(Title)`
+	margin-bottom: 3.2rem;
+`;
+
+const CancelButton = styled(Button)`
```

Comment:
Aren't those common buttons used in some screens? If so, we should just turn it into a component in order to make it reusable...

Reply:
@lcnogueira Yes, these buttons are used in a bunch of screens, but we decided to refactor it after releasing, with interface refactor too. That's why I'm reusing it from other screens.

Reply:
We talked about that when me and @mateus4k were developing a screen and needed to use the same button styles, but it didn't exist yet. We agreed to continue and refactor later 😄

Reply:
Ok, got it. Makes sense 👍